applications/DEMApplication/custom_constitutive/DEM_D_Linear_Simple_Coulomb_CL.cpp

```diff
@@ -0,0 +1,131 @@
+//  Kratos Multi-Physics - DEM Application
+//
+//  License:       BSD License
+//                 Kratos default license: kratos/license.txt
+//
+//  Main authors:  Rafael Rangel (rrangel@cimne.upc.edu)
+//
+// References using this model:
+// R.L. Rangel et al. (2024). Multiscale data-driven modeling of the thermomechanical behavior of granular media with thermal expansion effects. Comput Geotech, 176:106789.
+// R.L. Rangel et al. (2024). A continuum--discrete multiscale methodology using machine learning for thermal analysis of granular media. Comput Geotech, 168:106118.
+// S. Zhao et al. (2020). Multiscale modeling of thermo-mechanical responses of granular materials: A hierarchical continuum--discrete coupling approach. CMAME, 367:113100.
```

Comment:
In the paper by Zhao, although they said that the simple linear contact model was used, they did not explicitly give the formulation about how to calculate the normal and tangential stiffness. Should this one be replaced with another piece of literature?

Reply:
That's true. I changed the reference.